Sources/Sentry/SentrySDK.m

```diff
@@ -146,6 +148,9 @@ + (void)startWithOptions:(SentryOptions *)options
     [SentrySDK setCurrentHub:[[SentryHub alloc] initWithClient:newClient andScope:scope]];
     SENTRY_LOG_DEBUG(@"SDK initialized! Version: %@", SentryMeta.versionString);
     [SentrySDK installIntegrations];
+
+    [SentryCrashWrapper.sharedInstance startBinaryImageCache];
+    [SentryBinaryImageCache.shared start];
```

Comment:
`m`: I would prefer if the SentryCrashIntegration keeps this responsibility, cause it needs it if I'm not mistaken.

Reply:
SentryCrashIntegration may be disabled, then symbolication would be doomed.